### PR TITLE
Br2

### PR DESCRIPTION
**Pull request by UmangLadha**
Done some changes in component files and add some CSS.

### CODE REVIEW

**Comment by aadharbhatt** [comment]
@UmangLadha Avoid approving your own PR's and merging them without approval

File: creations/.eslintrc.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "rules": {
       "react/prop-types": 0,
       "indent": ["error", 4],
-      "linebreak-style": 1,
+      // "linebreak-style": 1,
```
Comment (aadharbhatt) [inline creations/.eslintrc.json:12]: @UmangLadha  Why is this commented ?
What problem was it causing ?

File: creations/package-lock.json [hunk 2]
```diff
@@ -1,16242 +1,8 @@
 {
```
Comment (aadharbhatt) [inline creations/package-lock.json]: @UmangLadha Why are these deleted?

File: creations/src/App.js [hunk 3]
```diff
@@ -15,12 +15,12 @@ function App() {
                 <Slider />
                 <nav className="navigation">
                     <ul className="list-non-bullet list">
-                        <li className="list-inline"><a className="toplink" href="http:#"> SERVICE </a></li>
-                        <li className="list-inline"><a className="toplink" href="http:#"> CLIENT </a></li>
-                        <li className="list-inline"><a className="toplink" href="http:#"> REVIEW </a></li>
-                        <li className="list-inline"><a className="toplink" href="http:#"> GALLERY </a></li>
-                        <li className="list-inline"><a className="toplink" href="http:#"> ABOUT US </a></li>
-                        <li className="list-inline"><a className="toplink" href="#contact"> CONTACT </a></li>
+                        <li className="list-inline"><a className="toplink" href="#Service-part"> SERVICES </a></li>
```
Comment (aadharbhatt) [inline creations/src/App.js:18]: @UmangLadha create custom functional component to accommodate these redundant changes

File: creations/src/components/Clients.js [hunk 4]
```diff
@@ -2,19 +2,167 @@ import React from "react";
...
     return (
-        <div>
+        <div id="#Cleint-part">
+            <h1 className="heading">CLIENTS</h1>
             <Figure className='details'>
-                <Figure.Image
-                    width={350}
-                    height={350}
-                    alt="350x350"
+                <Figure.Image
```
Comment (aadharbhatt) [inline creations/src/components/Clients.js:31]: @UmangLadha Create a JS object and iterate over it, passing its value to Figure.Image
Done write unnecessary code

File: creations/src/components/Services.js [hunk 5]
```diff
@@ -6,32 +6,35 @@ import logo from "../images/logo.jpg";
...
-                src={logo}
-            />
-            <Figure.Image
-                width={350}
-                height={350}
-                alt="171x180"
-                src={logo}
-            />
-        </Figure>
+        <div  id="#Service-part">
```
Comment (aadharbhatt) [inline creations/src/components/Services.js:9]: @UmangLadha Same here use object

File: creations/.eslintrc.json (hunk 1 above)
Comment (UmangLadha) [reply creations/.eslintrc.json:12]: it's giving the problem of semicolon I have solved reading this article.
https://eslint.org/docs/rules/linebreak-style#using-this-rule-with-version-control-systems

Comment (aadharbhatt) [reply creations/.eslintrc.json:12]: Although semicolon are good to have, its okay for now.
